### PR TITLE
change headline to welcome Hugo

### PR DESCRIPTION
**Pull request by hugo-ventil**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js" integrity="sha384-cVKIPhGWiC2Al4u+LWgxfKTRIcfu0JTxR+EQDz/bgldoEyl4H0zUF0QKbrJ0EcQF" crossorigin="anonymous"></script>
 
-    <title>Hello, world!</title>
+    <title>Hello, this one comes from Hugo</title>
```
Comment (jakub-sdlk) [inline index.html:14]: Why is this here?

Comment (hugo-ventil) [reply index.html:14]: Because I want to

**Review by jakub-sdlk — COMMENTED** [review]
Ok

**Review by jakub-sdlk — APPROVED** [review]
